src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -18,6 +18,9 @@
  *
  * @internal
  *
+ * @method void       setSerializedPath(array $serializedPath = null)
```

Comment:
Just wondering, if that interface is internal, why not add these methods directly in the interface instead of using phpdoc?